renderer-macos/lib/stubs/BriskCocoa.c

```diff
@@ -20,6 +16,6 @@ void brisk_caml_call(value f) {
   caml_release_runtime_system();
 }
 
-void retainView(NSView *view) { [retainedViews addObject:view]; }
+void retainView(NSView *view) { [view retain]; }
```

Comment:
how come does it work? It is invalid with ARC.

Reply:
🤷‍♀️